### PR TITLE
created header function

### PR DESCRIPTION
**Pull request by draganaaz**
- Exported cors settings to a function

### CODE REVIEW

File: auction9-backend/handler.js [hunk 1]
```diff
@@ -10,6 +10,13 @@ const mysql = require('serverless-mysql')({
   }
 });
 
+const header = () => {
+  return {
```
Comment (SvetislavMarjanovic) [inline auction9-backend/handler.js:14]: I thought to make a function for example generateResponse that receive statusCode and body, header will be preset there. But this one also looks fine for me.

**Review by SvetislavMarjanovic — APPROVED** [review]

File: auction9-backend/handler.js (hunk 1 above)
Comment (draganaaz) [reply auction9-backend/handler.js:14]: @SvetislavMarjanovic  Sure, I will do that once I finish history.

File: auction9-backend/handler.js [hunk 2]
```diff
@@ -10,6 +10,17 @@ const mysql = require('serverless-mysql')({
   }
 });
 
+const generateResponse = (statusCode, body) => {
+  return {
+    statusCode: this.statusCode,
```
Comment (SvetislavMarjanovic) [inline auction9-backend/handler.js:15]: `this` is not needed for parameters that are passed to the method.
Remove that and everything else is good.

**Review by mjovanovic — APPROVED** [review]

File: auction9-backend/handler.js (hunk 2 above)
Comment (draganaaz) [reply auction9-backend/handler.js]: Removed.

**Pull request merged**